packager/packager.cc

```diff
@@ -657,10 +657,10 @@ Status CreateAudioVideoJobs(
       }
 
       std::vector<std::shared_ptr<MediaHandler>> handlers;
-      if (is_text) {
+      /*if (is_text) {
         handlers.emplace_back(std::make_shared<TextPadder>(
```

Comment:
This is a temporary fix that needs more evaluation. The TextPadder is not needed for live, and it mixes timescales...